### DISCUSSION TITLE
Creating round edges on images

### DISCUSSION THREAD

**Discussion by Hrovatin** [cotes2020/jekyll-theme-chirpy#2103] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Is there a way to make images with rounded edges, such as border-radius? 

### Operations you have already tried

I tried adding border-radius paraeter to the curly brackets used to define the image size but that did not work.

**Comment by SoumyaK4** [comment]
Try using custom scss [here](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/assets/css/jekyll-theme-chirpy.scss)

**Reply by Hrovatin** [reply]
Thank you for your suggestion. I downloaded the file and put it in `assets/css/` and added:

```
.rounded {
  border-radius: 7px;
}
```

Then I use it in my image as 
```
![Desktop View](ambient_expression.jpg){: width="800" alt="abc" .shadow .rounded;}
```

Any ideas on what else may be needed to make it appear in the image?

**Reply by SoumyaK4** [reply]
@Hrovatin as of my knowledge , kramdowns don't need the ending ';' 
so I think it should be
```
![Desktop View](ambient_expression.jpg){: width="800" alt="abc" .rounded}
```

**Reply by Hrovatin** [reply]
Sadly this did not work but tis' answer resolved it. I assume that somehow the scss does not get incorporated in the final style used.

**Comment by tis-starlight** [comment, marked as answer]
``` 
![Desktop View](ambient_expression.jpg){: width="800" alt="abc" style="border-radius: 7px;"}
```

**Reply by Hrovatin** [reply]
This resolved it, thank you.
